src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -701,4 +701,7 @@ Valid from:</comment>
     <value>The package '{0}' contains an entry which is unsafe for extraction.</value>
     <comment>{0} is the package identity.</comment>
   </data>
+  <data name="NoRepositoryCountersignature" xml:space="preserve">
+    <value>The package signature did not have a repository countersignature.</value>
```

Comment:
Can we improve the error message to be more meaningful? Some context around why is this a problem will be helpful to the user.